### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by disc3110**
Created an App with that keeps track of the user's expenses and helps with his budget.
The app consists of three views, a Login Screen, a Category screen, and a Transaction screen.

Here is a briefly explanation of the project: https://www.loom.com/share/b8a15d9415034fa7a7bbccde287c1ae4

Don't forget to change the user and password in config/database.yml, to have some test data `run rails db:seed`

### CODE REVIEW

File: app/views/deals/_deal.html.erb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<div class="m-2 bg-white border d-flex justify-content-between p-2" id="<%= dom_id deal %>">
+  <p>
+    <%= deal.name %>
+  </p>
+
+  <p>
+    <%= deal.amount %>
+  </p>
+
```
Comment (Dipeshtwis) [inline app/views/deals/_deal.html.erb]: - [x] In this page, Please show the date when the transaction happens as per the design.

### Your design
Screenshot from 2022-04-29 23-57-50


### GIven design
Screenshot from 2022-04-30 00-16-20

File: app/views/deals/_form.html.erb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+          <li><%= error.full_message %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
+
+  <%= form.text_field :name, class:'form-field', placeholder:'Name' %>
+
+  <%= form.text_field :amount, class:'form-field', placeholder:'Amount' %>
+
```
Comment (Dipeshtwis) [inline app/views/deals/_form.html.erb:17]: - [x] I would recommend adding the feature to select the `categories` here in the form as per requirements. Please update this.

Screenshot from 2022-04-30 00-17-18

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @disc3110, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

# HIghlights
:+1: The project is deployed online.
:+1: All tests are running. :heavy_check_mark: 

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N.A.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: app/views/deals/_deal.html.erb [hunk 3]
```diff
@@ -1,7 +1,8 @@
 <div class="m-2 bg-white border d-flex justify-content-between p-2" id="<%= dom_id deal %>">
-  <p>
-    <%= deal.name %>
-  </p>
+  <div class="d-flex flex-column justify-content-between">
+    <p><%= deal.name %></p>
+    <p class="fs-075r"><%= deal.created_at.strftime("%d %b %Y")%></p>
```
Comment (rukundoeric) [inline app/views/deals/_deal.html.erb:4]: - [ ] <b>OPTIONAL:</b> Remember to deploy so that your latest changes can be reflected on your deployed app.

**Review by rukundoeric — APPROVED** [review]
Hi @disc3110,

_Excellent job on this project. Keep it up 🎆_

_Congratulations! 🎉  on the splendid job 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_


Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

------
>As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**